inst/qml/JAGS.qml

```diff
@@ -122,6 +123,379 @@ Form
 		}
 	}
 
+	Section
+	{
+		title: qsTr("Customizable Inference")
+
+		TabView
+		{
+			id:					customizablePlots
+			name:				"customInference"
+			maximumItems:		10
+			newItemName:		qsTr("Plot 1")
+			optionKey:			"customInferenceOptions"
+			Layout.columnSpan:	2
+
+			content: Group
+			{
+
+				Group // Parameter selection
+				{
+					indent: true
+					columns: 1
+
+					Group
+					{
+						columns: 2
+						Group
+						{
+							title: qsTr("Parameter selection")
+
+							DropDown
+							{
+								label: 				qsTr("Parameter")
+								name: 				"customInferenceParameter"
```

Comment:
```suggestion
								name: 				"parameter"
```
since this is a part of the `customInference` list, you don't need to prefix any of the suboptions like that